wiki/Ranking_Criteria/en.md

```diff
@@ -17,12 +17,13 @@ Glossary
 
 ### Mapset
 
-- **Drain time:** The time during which the health-bar drains. Excludes break periods. Taiko mode’s drain time includes sliders and the intended length of spinners. 
-- **Reasonable Spread:** A mapset without drastically large differences between difficulties as dictated by difficulty-specific rules and guidelines. 
+- **Drain time:** The time during which the health-bar drains. Excludes break periods. osu!taiko's drain time includes sliders and the intended length of spinners. 
```

Comment:
`Drain time` -> `Drain Time`?

Should the terms be always first letter uppercased? Will only mention this term in this review.

Reply:
most terms only capitalize the first word's letter, so went with that throughout